cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -691,6 +695,22 @@ public static void initPackageByTemplate(Path path, String packageName, String t
             Path source = path.resolve("lib.bal");
             Files.move(source, source.resolveSibling(guessPkgName(packageName, template) + ".bal"),
                     StandardCopyOption.REPLACE_EXISTING);
+        } else if (template.equalsIgnoreCase("graphql-client")) {
+            // project_name/
```

Comment:
Shall we change this to `package_name`

Reply:
Also, don't we generate a source `.bal ` for this template? A package with no source files or test files is considered an empty package and the only scenario where an empty package is allowed is when it has a compiler plugin. 

For all other scenarios, `bal build` and `bal pack` will fail with an error. We need to have a source `.bal` file added to the package.

Reply:
We have a separate `graphql` command to do the graphql client generation.
https://ballerina.io/learn/graphql-client-tool/#execute-the-graphql-cli-command

We are planning to use this `bal new <package_name> -t graphql-client` command as a prerequisite to generate the templates. Should we still proceed and add a .bal file? If so what should be the content of this file?